Frontend/CO.CDP.OrganisationApp/Pages/Shared/PaginationPartialModel.cs

```diff
@@ -0,0 +1,27 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace CO.CDP.OrganisationApp.Pages.Shared
+{
+    public class PaginationPartialModel
+    {
+        [Required]
+        public int CurrentPage { get; set; }
+
+        [Required]
+        public int TotalItems { get; set; }
+
+        [Required]
+        public int PageSize { get; set; }
+
+        [Required]
+        public string OrganisationType { get; set; } = "buyer";
```

Comment:
Does OrganisationType need to be on this model? The rest of it looks generic. And I can't see that it's used in the template?